mobile/ios/ios-device.ts

```diff
@@ -320,11 +327,130 @@ export class IOSDevice implements Mobile.IIOSDevice {
 			installationProxy.deployApplication(packageFile).wait();
 			installationProxy.closeSocket();
 		}).future<void>()();
+    }
+
+    private openSafariFrontEnd(packageName: string): void {
+        var tnsIosPackage = this.$npm.install("tns-ios").wait();
+        var safariPath = path.join(tnsIosPackage, "WebInspectorUI/Safari/Main.html");
+        this.$childProcess.exec("open -a Safari " + safariPath).wait();
+    }
+
+    private attachWhenReady(packageName: string,
+        notificationProxyClient: iOSProxyServices.NotificationProxyClient): void {
+
+        var identifier = this.getIdentifier();
+        this.$logger.info("Device Identifier: " + identifier);
+
+        var appReadyForAttachMessage = packageName + ":NativeScript.Debug.ReadyForAttach";
+        notificationProxyClient.addObserver(appReadyForAttachMessage, () => {
+            this.$logger.info("Got ReadyForAttach");
+
+            var devicePort = 8080;
+            var localPort = 8080;
```

Comment:
Add a comment that we are planning to allow changing them in the future and we have a deal.